### PR TITLE
Add docs on running checkstyle locally

### PR DESCRIPTION
**Pull request by grantatspothero**
Just helpful for new people like me. Thanks @jirassimok for the tip.

### CODE REVIEW

**Review by losipiuk — APPROVED** [review]
thx

File: .github/DEVELOPMENT.md [hunk 1]
```diff
@@ -43,8 +43,11 @@ much easier and reduces the chance of introducing unintended changes in behavior
 
 We recommend you use IntelliJ as your IDE. The code style template for the
 project can be found in the [codestyle](https://github.com/airlift/codestyle)
-repository along with our general programming and Java guidelines. In addition
-to those you should also adhere to the following:
+repository along with our general programming and Java guidelines. 
+
+To run checkstyle checks before opening a PR: `./mvnw checkstyle:check@checkstyle`
```
Comment (hashhar) [inline .github/DEVELOPMENT.md:48]: Does `./mvnw validate` seem to do what you want? It'll catch more things (if the goal is to make sure people submit "working" PRs).

**Review by hashhar — COMMENTED** [review]
I personally think people should run with `clean verify/install` on the modules they touch at the very least. (And maybe errorprone too?).

But this doc is better than what we had before anyway, thanks.

File: .github/DEVELOPMENT.md (hunk 1 above)
Comment (grantatspothero) [reply .github/DEVELOPMENT.md:48]: Oh that is even better since it runs checkstyle + dep enforcer and other things 👍

**Review by hashhar — APPROVED** [review]

**Pull request merged**